syntax = "proto3";

package greeting;

message ChatMessage {
  string username = 1;
  string text = 2;
  string emoji = 3;
}

service ChatService {
  rpc ChatStream (stream ChatMessage) returns (stream ChatMessage);
}